apps/web/i18n/lang/de.json

```
@@ -1045,59 +1036,44 @@
       "submitWunschpaketServices": "Einstellungen für Wunschzustellung-Services übernommen.",
       "apply": "Übernehmen",
-      "parcelApiCallLog": "Parcel API call log",
       "invalidPostalCodeMessage": "Bitte prüfen Sie die Eingabe der Postleitzahl. Es wird ein 5-stelliger numerischer Wert erwartet - z.B. '34117'.",
-      "noServicesMessage1": "Bitte prüfen Sie die Eingabe der Postleitzahl. Es wird ein 5-stelliger numerischer Wert erwartet - z.B. '34117'.",
-      "noServicesMessage2": "Es konnten keine gültigen Services für DHL Wunschzustellung ermittelt werden.",
-      "assistantDescription": "Verwende diesen Assistenten, um mit dem DHL Wunschzustellung-Plugin die DHL Wunschzustellung-Services in deinem Shop zu integrieren.",
-      "warning": "Warnhinweis",
-      "wunschpaketRequirementWarning": "Speichere einen Aufpreis für den Service {preferredDayMsg}. Die Wunschzustellung-Services selbst aktivierst du dann im Assistenten {shippingSettingsMsg}. Bestimme hier außerdem die Cut-off-Zeit und lege fest, an welchen Tagen keine Paketübergabe an DHL erfolgt.",
-      "additionalCharge": "Aufpreis",
-      "preferredDay": "Aufpreis Liefertag",
-      "cutofftime": "Cut-off-Zeit",
-      "cutofftimeDescription": "Die Cut-off-Zeit ist die spätestmögliche Bestellzeit, bis zu dem der Mindestwunschtag (Bestelltag + 2 Werktage) garantiert werden kann. Wenn der Zeitpunkt überschritten wird, wird der frühestmögliche Liefertag, der im Frontend angezeigt wird, einen Tag nach hinten verschoben (Bestelltag + 3 Werktage).",
-      "hour": "Stunde",
-      "minute": "Minute",
-      "exclusionTransfer": "Ausschluss von Übergabetagen",
-      "exclusionTransferDescription": "An den Werktagen, die du aktivierst, findet keine Übergabe von Paketen an DHL statt (zusätzlich zu Sonntagen und Feiertagen). Diese Angabe ermittelt den ersten Liefertag, der im Webshop angezeigt wird.",
-      "step1Title": "Service-Einstellungen",
-      "gmapsApiKey": "API-Schlüssel",
-      "additionalChargeDescription": "Gib den Bruttopreis für den Service {preferredDayMsg} ein. Der Preis wird deinen Kund:innen in Rechnung gestellt, wenn sie den Service in deinem Webshop wählen. Gib 0 ein, um diesen Service kostenfrei anzubieten.",
-      "postnumberRequired": "Möchtest du, dass die DHL Postnummer ein Pflichtfeld für deine Kunden ist, wenn sie eine Postfiliale im Finder wählen?",
-      "finderSettingsTitle": "Einstellungen für den DHL Packstation/Postfiliale-Finder",
-      "finderSettingsDescription": "Gib deinen Google Maps API-Schlüssel ein, um den Finder korrekt verwenden zu können. Im Assistenten {shippingSettingsMsg} kannst du den Finder aktivieren. Entscheide außerdem, ob die DHL Postnummer ein Pflichtfeld sein soll, wenn ein Kunde im Finder eine Postfiliale wählt."
+      "additionalCharge": "Aufpreis"
     },
     "packstation": {
       "title": "DHL Packstation / Postfiliale",
+      "zipcodeLabel": "Geben Sie Ihre Postleitzahl ein",
```

```suggestion
      "zipcodeLabel": "Postleitzahl",
```
Otherwise it breaks the Ui text is just too long